lab3/lab3_task_1.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8"/>
+    <title>Lab3_1</title>
+    <style>
+        p:first-of-type {
+            font-size: larger;
+        }
+        p:last-of-type {
+            color: #999
```

Comment:
Good! In accordance with https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation
BTW, go over the whole guide, contains some good practices as seen by Google. If you do something like this guide says - you can always refer to it.